### PR TITLE
Fix stack_trace_unittest for cobalt.

### PR DESCRIPTION
**Pull request by ghost**
b/316403754

### CODE REVIEW

File: base/debug/stack_trace_unittest.cc [hunk 1]
```diff
@@ -157,7 +162,8 @@ TEST_F(StackTraceTest, DebugOutputToStreamWithNullPrefix) {
 #endif  // !defined(__UCLIBC__) && !defined(_AIX)
 
 #if BUILDFLAG(IS_POSIX) && !BUILDFLAG(IS_ANDROID)
-#if !BUILDFLAG(IS_IOS)
+// Starboard does not support relative file paths, needed by |SpawnChild()|.
+#if !BUILDFLAG(IS_IOS) && !defined(USE_HACKY_COBALT_CHANGES)
```
Comment (sherryzy) [inline base/debug/stack_trace_unittest.cc:166]: If it's not supported by Starboard, we should use !defined(STARBOARD)?
I'm not so sure when should we use this `USE_HACKY_COBALT_CHANGES`, @andrewsavage1 to clarify this?

Comment (andrewsavage1) [reply base/debug/stack_trace_unittest.cc:166]: I added `USE_HACKY_COBALT_CHANGES` for changes we want to clean up later but want to get working now

File: base/process/process_starboard.cc [hunk 2]
```diff
@@ -43,6 +43,14 @@ bool Process::IsProcessBackgrounded() const {
   return false;
 }
 
+bool Process::IsValid() const {
+  return true;
+}
+
+bool Process::WaitForExitWithTimeout(TimeDelta timeout, int* exit_code) const {
+  return false;
```
Comment (andrewsavage1) [inline base/process/process_starboard.cc:51]: Should there be a NOTIMPLEMENTED called here?

File: base/test/BUILD.gn [hunk 3]
```diff
@@ -300,6 +300,12 @@ static_library("test_support") {
     ]
   }
 
+  if (!use_blink && is_starboard) {
+    sources += [
+      "multiprocess_test.cc",
```
Comment (andrewsavage1) [inline base/test/BUILD.gn:305]: Starboard is always single-process, so I don't think we should be running this test

File: base/process/launch_starboard.cc [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+#include "base/process/launch.h"
+
+namespace base {
+
+void RaiseProcessToHighPriority() {
+  // Impossible on iOS. Do nothing.
+}
+
+bool GetAppOutput(const CommandLine& cl, std::string* output) {
```
Comment (andrewsavage1) [inline base/process/launch_starboard.cc:13]: Should any of these call NOTIMPLEMENTED? Or is this how we want them long-term?

**Review by andrewsavage1 — APPROVED** [review]

**Pull request merged**